migration.md

```diff
@@ -0,0 +1,87 @@
+
+### Connection profiles
+
+The Fabric Gateway client API does not use common connection profiles. Instead, only the endpoint address of the Fabric Gateway service is required to establish a gRPC connection that will be used when connecting a Gateway instance. Since the Fabric Gateway service is provided by network peers, the endpoint address may be one of the peer addresses that would be defined in a connection profile. It could also be the address of a load balancer or ingress controller that forwards connections to network peers, providing high availability.
+
+### Wallets
+
+The legacy SDKs provide wallets for credential management. Wallets perform two functions:
+
+1. Persistent credential storage.
+1. Configuration of the Gateway client based on the type of credentials (for example, identities managed by a Hardware Security Module managed).
+
+Using the Fabric Gateway client API, the mechanism for storing credentials is a choice for the client application. The application may continue using the legacy SDKs to access credentials stored in a wallet, or may use a different mechanism for storing and accessing credentials.
+
+To connect a Gateway instance, the application simply provides an Identity object and a signing implementation. Helper functions are provided to create an Identity object from an X.509 certificate, and also to create a signing implementation from either a private key or an HSM-managed identity. To make use of alternative signing mechanisms, the application may provide its own signing implementation.
+
+### Endorsement requirements
+
+When using the legacy SDKs in more complex scenarios, such as those involving private data collections, chaincode-to-chaincode calls, or key-based endorement policies, it is often necessary for the client application to explicitly specify endorsement requirements for a transaction invocation. This may be in the form of specifying chaincode interests, endorsing organizations, or endorsing peers.
+
+Using the Fabric Gateway client API, it is generally not necessary for the client application to specify any endorsement requirements. The Fabric Gateway service dynamically determines the endorsement requirements for a given transaction invocation and uses the most appropriate peers to obtain endorsement.
+
+In order to ensure that private data is not distributed to organizations that should not have access to it, there are two notable scenarios that do require the application to explicitly specify the organizations that may be used for endorsement:
+
+1. Proposals containing transient data where the Fabric Gateway service's organization is unable to endorse the transaction proposal.
+1. Invocation of transactions that do blind writes to private data collection from which they do not have read persmission.
```

Comment:
Again, this is only when transient data is sent in the proposal

Reply:
Reworked the wording to say that both scenarios apply only to transaction proposals that contain transient data